src/Nox.Lib/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Nox.Extensions;
+
+internal static class StringExtensions
+{
+    public static string StripQueryParameters(this string input) => input.Split('?')[0];
```

Comment:
this approach will require less memory allocation?
```suggestion
    public static string StripQueryParameters(this string input) =>input.Substring(0, input.IndexOf(',?'));
```